clusterloader2/README.md

```diff
@@ -1,21 +1,10 @@
 # ClusterLoader
 
-## Running ClusterLoader
+Clusterloader is a tool that allows you to deploy multiple kubernetes objects to your cluster and measure performance metrics.
```

Comment:
Let's spice it up a bit :) Something like:

Cl2 is a "bring your own yaml" Kubernetes load testing tool being an official K8s scalability and performance testing framework. 
<And here we can given an api overview mentioned in some other comment>

The CL2 tests are written in yaml using the declarative paradigm. A test defines a set of states in which a cluster should be (e.g. I want to run 10k pods, 2k cluster-ip services, 5 daemon-sets, etc.) and CL2 transitions the cluster through that states. In addition, the test also specifies how the transitions should happen (e.g. pod throughput) and which performance characteristics should be measured during the execution (link to the measurement list section here). Last but not least, CL2 provides an extra observability of the cluster during the test (link to promethus section here).

The CL2 test API is explained here (link).

Reply:
Done.